Change extension workshop urls when running dev

feugy opened the pull request:
Fix #7830

---

## Description

When running `yarn dev:amo`, the link to extension workshop in footer now point at ttps://extensionworkshop-dev.allizom.org.
Please note the url is not available :cry: 

~I've also added `.gitattribute` file to force LF end of lines on checkout. Developers with `auto.crlf=true` in their global git configuration. One cannot run the app with CLRF in script files.~

## Screenshot

![image](https://user-images.githubusercontent.com/186268/55667633-09bc8d00-585f-11e9-8236-e0a085f9cc58.png)

willdurand commented:
@feugy not sure you receive notifications when a review has been made. Thanks for your work. I requested a minor change though :)

feugy commented:
@willdurand I got the notifs, I was just busy :smile:

I've removed the .gitattributes file and updated the PR description. 
I reckon that very few people will checkout the code on a NTFS partition. It's could be something to consider though, since Windows now have "real" bash and could run .sh files.

willdurand commented:
Yep, that's right but we try not to have too many generic files here. I believe one could configure windows globally instead.

caitmuenster commented:
Thanks so much for the patch, @feugy! Your contribution has been added to our [recognition wiki ](https://wiki.mozilla.org/Add-ons/Contribute/Recognition)and your mozillians.org profile has been vouched. 🎉

Welcome onboard! I look forward to seeing you around the project. :)

feugy commented:
Thank you @caitmuenster & @willdurand!
